cupy/testing/_array.py

```diff
@@ -103,7 +107,8 @@ def assert_array_equal(x, y, err_msg='', verbose=True, strides_check=False):
             raise AssertionError('\n'.join(msg))
 
 
-def assert_array_list_equal(xlist, ylist, err_msg='', verbose=True):
+def assert_array_list_equal(xlist, ylist, err_msg='', verbose=True,
```

Comment:
This is not NumPy-compatible API, so I think `strict` options is not needed here.

Reply:
Got it. I have revert the addition of `strict` options here so this function remains untouched.